packages/l2b/src/implementations/discovery-ui/types.ts

```diff
@@ -2,7 +2,7 @@
 
 export type ApiProjectsResponse = ApiProjectEntry[]
 
-export interface ApiProjectEntry {
+interface ApiProjectEntry {
```

Comment:
And again here IMO going too far. Files that define types should export everything whether it's used or not.